src/term.rs

```diff
@@ -513,23 +527,20 @@ fn show_precedence_cla(
     match term {
         Var(0) => "undefined".to_owned(),
         Var(i) => {
-            if depth >= *i as u32 {
-                from_u32(depth + 97 - *i as u32)
-                    .expect("error while printing term")
-                    .to_string()
+            let cast = *i as u32;
+            let ix = if cast <= depth {
```

Comment:
are you sure about `<=` and not `<` here? the reversal of the original direction of the comparison would be the latter

Reply:
`<=` is correct. When `depth == i`, the first branch evaluates to 0, and the second evaluates to `max_depth - 1`. We want the zero - it corresponds to an `a`, the outermost variable in any abstraction chain.